### PR TITLE
Fix balances

### LINKED ISSUE #1007: Add total_limbo_balance to "Closing" sum on channels page
We should add `total_limbo_balance` it to the Closing sum at the top of the channel page on desktop: https://api.lightning.community/#pendingchannels

### LINKED ISSUE #1016: Pending-open balance is being added to unifued balance twice
We need to remove `pendingBalanceSatoshis` from ComputedWallet.totalBalanceSatoshis()

https://github.com/lightninglabs/lightning-app/blob/4c586796ca9bf4ee2cbb585178fa81cb252691dd/src/computed/wallet.js#L19-L32

### PR DESCRIPTION
**Pull request by tanx**
Closes #1007, closes #1016

### CODE REVIEW

File: src/action/channel.js [hunk 1]
```diff
@@ -168,7 +168,6 @@ class ChannelAction {
         status: 'waiting-close',
       }));
       this._store.pendingChannels = [].concat(pocs, pccs, pfccs, wccs);
-      this._store.limboBalanceSatoshis = response.totalLimboBalance;
```
Comment (valentinewallace) [inline src/action/channel.js]: Ah so [this](https://github.com/lightninglabs/lightning-app/issues/1007#issuecomment-472722390) comment is outdated? Basically we were calculating correctly to begin with? I forget why we initially thought `limboBalance` was relevant to anything :thinking:

File: src/computed/wallet.js [hunk 2]
```diff
@@ -21,13 +21,13 @@ const ComputedWallet = store => {
         balanceSatoshis,
         pendingBalanceSatoshis,
         channelBalanceSatoshis,
-        limboBalanceSatoshis,
+        channelBalanceClosingSatoshis,
```
Comment (valentinewallace) [inline src/computed/wallet.js]: Cool, this does make sense to fix the [double-add problem](https://github.com/lightninglabs/lightning-app/issues/1016#issuecomment-472725500). Between this and `pendingBalanceSatoshis` we cover all the pending funds, then `balanceSatoshis` is on-chain proper, and `channelBalanceSatoshis` is off-chain proper.

File: src/computed/channel.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ const ComputedChannel = store => {
     get channelBalanceOpenSatoshis() {
       return (store.channels || [])
         .filter(c => c.active)
-        .map(c => Number(c.localBalance))
+        .map(c => c.localBalance)
```
Comment (valentinewallace) [inline src/computed/channel.js]: Good catch :+1:

File: src/action/channel.js (hunk 1 above)
Comment (tanx) [reply src/action/channel.js]: Yes. They are one and the same. We can calculate limboBalanceSatoshis from the pendingChannels grpc response array.

**Pull request merged**